qiime/beta_diversity.py

```diff
@@ -173,6 +171,7 @@ def single_file_beta(input_path, metrics, tree_path, output_dir,
                              % (metric, ', '.join(list_known_metrics())))
                 exit(1)
         if rowids is None:
+            otumtx = otu_table.matrix_data.T.toarray()
```

Comment:
This _probably_ can just use the row by row stuff done below. Fairly certain there is never a need to do a dense conversion here.